Makefile

```diff
@@ -170,6 +170,7 @@ build/install.sh:
 		cat $$script >> tmp/install.sh ; \
 	done
 	sed -n '/# Magic end/,$$p' scripts/install.sh | sed '1d' >> tmp/install.sh
+	sed -i 's/^DISABLE_REPORTING=.*/DISABLE_REPORTING=1/' tmp/install.sh # disable reporting
```

Comment:
This is going to disable reporting for prod.

Reply:
You could put `DISABLE_REPORTING=1` in the Manifest and then delete the whole line in `make build/templates/install.tmpl`